packages/frontend/src/components/info/Accordion.tsx

```diff
@@ -23,30 +23,47 @@ export const InfoAccordion = () => {
             <AccordionStyledTrigger heading="What is this?" />
           </StyledHeader>
           <StyledContent>
-            In an effort to make our ticket distribution more efficient and fair, we are selling a{' '}
-            <Italic>portion</Italic> of this year’s tickets via an on-chain Auction & Raffle. Typically we sell tickets
-            in waves; attendees need to wait for a specific release time and refresh the ticket shop rapidly in order to{' '}
-            <Italic>hope</Italic> to claim & checkout with a ticket. Not to mention the need for a speedy internet
-            connection and crossing your fingers that you’re close enough to our ticketing servers to be one of the
-            first to secure a ticket.
-            <br />
-            <br />
-            This year, we wanted to try something different, so we are experimenting with an on-chain Auction & Raffle
-            to sell a <Italic>portion</Italic> of Devcon tickets.
+            <span>
+              With the generous help of the{' '}
+              <Link href="https://x.com/fairyrnd" target="_blank">
+                Fairy
+              </Link>{' '}
+              &{' '}
```

Comment:
Should we escape the ampersands?
```suggestion
              &amp;{' '}
```
There are lots of them in the copy below